packages/core/cli.ts

```diff
@@ -53,183 +61,175 @@ yargs(process.argv.slice(2))
 			"An example is shown in the examples section of this help message.",
 		(yargs: Argv) => {
 			yargs
-				.positional("target", {
+				.positional("fuzz_target", {
```

Comment:
Could this be `fuzz_file`? To me `fuzz_target` and `target` are synonyms for the fuzz function.
In the end, this always points to a _file_ where the fuzz function / fuzz entry point will be.
If someone wakes me up at night and asks about the parameter to specify the file with the fuzz function, I won't know, unless it contains `file` in it!

I write a lot here because 1) this is a breaking change, and 2) I want the final result to be intuitive!

Reply:
We discussed naming inconsistencies quite a lot in past, as we would also need to change parameter, function, class names, etc.

As you said, this would be a breaking change, so I'm really hesitant to change it and would rather wait until we need other braking changes. Waiting could also introduce other problems, though. WDYT?

Reply:
The initial idea was to stay close to existing fuzzers and use the same parameter names. That turned out to be not ideal. This PR tries to _not_ introduce breaking changes, so the config files names are basically a given. We could think about renaming parameters in a more global and breaking way, though.